src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/CreateTask.groovy

```diff
@@ -75,6 +75,7 @@ class CreateTask extends AbstractServerTask {
                 params.put('template', server.template)
             }
             executeServerCommand(project, 'create', params)
+            createApplicationFolders()
```

Comment:
I don't know if it is ok to tie this to server creation. I thought we would going to detect that the apps or dropins folder did not exist during the installApps task instead. Could they run installApps against a server that was not created with our create task?

Reply:
Also, do we need to make similar changes to maven for this?